src/lib/models/board.ts

```diff
@@ -15,7 +15,7 @@ type PieceConstructor = {
 
 export default class Board {
   static LANE_SIZE = 8;
-  #board: Piece[][] = [];
+  #board: (Piece | null)[][] = [];
```

Comment:
good candidate for extraction

Reply:
```suggestion
  type BoardEntry = Piece | null
  type BoardGrid = BoardEntry[][]
  ... 
  #board: BoardGrid = [];
```

Reply:
Just want to discuss this first so leaving comment